scrub-logs.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python
+
+##===---------- scrub-logs.py - generate README from Wiki sources ---------===##
```

Comment:
Usually having dashes in Python file names is undesirable because you won't be able to use the file as a module. But I believe here it should be fine as it's not supposed to be used as a library.